src/promptflow/tests/executor/unittests/executor/test_tool_resolver.py

```diff
@@ -161,6 +162,22 @@ def test_resolve_tool_by_node_with_duplicated_inputs(self, resolver, mocker):
         assert isinstance(exec_info.value.inner_exception, NodeInputValidationError)
         assert "These inputs are duplicated" in exec_info.value.message
 
+    def test_resolve_tool_by_node_with_invalid_template(self, resolver, mocker):
+        node = mocker.Mock(tool=None, inputs={})
+        node.name = "node"
+        node.type = ToolType.PROMPT
+        mocker.patch.object(resolver, "_load_source_content", return_value="{{current context}}")
+
+        with pytest.raises(ResolveToolError) as exec_info:
+            resolver.resolve_tool_by_node(node)
+
+        assert isinstance(exec_info.value.inner_exception, TemplateSyntaxError)
+        expected_message = (
+            "Tool load failed in 'node': (TemplateSyntaxError) in line 1: "
```

Comment:
How about "Tool load failed in 'node' at line 2: (TemplateSyntaxError) expected token ... " to align with the one with generate tool meta?

Or maybe "Tool load failed in 'node'. Jinja parsing failed at line 2: (TemplateSyntaxError) expected token ... "